redaxo/src/core/lib/form/form_base.php

```diff
@@ -1059,7 +1064,8 @@ protected function validate()
                 }
 
                 $validator = $element->getValidator();
-                if (!$validator->isValid($element->getSaveValue())) {
+                $saveValue = $element->getSaveValue();
+                if (!$validator->isValid((string) $saveValue)) {
```

Comment:
Beispiel: Wir valideren jetzt also ob 5 eine „5“ ist und speichern dann eine (int) 5 ?

Reply:
Also wenn jemand dem Validator zum Beispiel `->add('min', '...', 6)` hinzugefügt hat, dann wandeln wir hier die 5 in einen String, weil der Validator das so möchte, letztlich wird es aber in der `min()`-Methode vom Validator implizit wieder zurückgewandelt für den `>=`-Check.

Reply:
Ob das jetzt gut ist, oder ob man es anders lösen sollte, da bin ich mir wie gesagt unsicher.

Reply:
Ich denke - so wie es jetzt ist - ist es ohne BC Break.. daher passt das für mich